controller/transfer/project_transfer_manager.py

```diff
@@ -129,14 +130,14 @@ def import_sample_project(
         )
 
         notification.send_organization_update(
-            project_item.id, "project_update", is_global=True
+            project_item.id, f"project_update:{str(project_item.id)}", is_global=True
         )
         return project_item
 
 
 def import_file(
     project_id: str,
-    user_id: str,
+    import_user_id: str,
```

Comment:
user_id has been used in lines 494 and 827 and needs to be replaced there with import_user_id.
Don't throw errors currently, because user_id is defined somewhere else in the method, but this is not intended.